.github/workflows/main_discordbotsoftwire.yml

```diff
@@ -0,0 +1,37 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
```

Comment:
Should this be included in this PR? 🙂

Reply:
I think that sneaked in here when I merged main into this PR, but now main has been reverted so I'll get rid of that.

Reply:
Main has been reverted? Why?? There should never be any force push to main.